README.md

```diff
@@ -31,7 +31,9 @@ $ git clone https://github.com/hashrocket/tilex
 $ cd tilex
 $ mix deps.get
 $ mix ecto.setup
+$ cd ./assets
```

Comment:
Great catch! Thank you for taking the time to contribute!

Is there a reason to `cd ./assets`, rather than just `cd assets`?

Reply:
Old habits die hard. Changed in 13670c5